nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/DependencyConflictResolver.cs

```diff
@@ -0,0 +1,680 @@
+using System.Collections.Immutable;
+using System.Text;
+using System.Text.RegularExpressions;
+
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Frameworks;
+using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+using NuGetUpdater.Core.Analyze;
+
+using Task = System.Threading.Tasks.Task;
+
+namespace NuGetUpdater.Core;
+
+// Data type to store information of a given package
+public class PackageToUpdate
+{
+    public string packageName { get; set; }
```

Comment:
Make public members PascalCased, e.g., `public string PackageName { get; set; }`.  Same for other properties.